File: [프로그래머스 Al]/이진 변환 반복하기.py
def solution(s):
    from collections import Counter
    zero,cnt = 0,0
    
    while s != '1':
        cnt += 1
        zero += Counter(s)['0']
        s = str(bin(int(Counter(s)['1']))).replace("0b","")
        
    
    return [cnt,zero]

File: [프로그래머스 Al]/이진 변환 반복하기.js
function solution(s) {
    let zero = 0
    let cnt = 0
    
    while(s !== '1'){
        const temp = [...s].reduce((total,string)=>{
          if(string === '0'){
              zero += 1
              return total
          }  
            return total +1
        },0)
        s = temp.toString(2)
        cnt += 1
    }
    
    return [cnt,zero];
}
